### ISSUE TITLE
Integrate normal bows into AI and Captain Guards/Kings 

### ISSUE THREAD

**Issue by Lincolnium** [seymourimadeit/guardvillager-suggestions#124]

**Comment by omeggga** [comment]
I approve of normal bows, as for captain guards or kings... why? What do you want them to do exactly?

**Comment by Lincolnium** [comment]
I would like to see one captain guard per village, and the captain villager would have more health, and he generally stronger. A really far-fetched idea would be to make the captain hirable, and make it so they would follow you for a certain amount of time, or this could just be a completely different villager mob. As another suggestion, I would LOVE to see the villagers be compatible with guns without roses or CGM, as I think it would be fun to use and mess around with.

**Comment by omeggga** [comment]
But you can already do this by giving one single guard villager the best equipment.

**Comment by Lincolnium** [comment]
Yeah, perhaps so, but maybe a villager banner on the captains head? But I would love to see more compatibility with other modded weapons, that are not bows, crossbows, or melee weapons. It’s super cool how the villagers use bucklers from big brain, so if they could do that with other weapons, that would be cool.
